src/shared/sol-random.c

```diff
@@ -86,7 +86,9 @@ getrandom_shim(void *buf, size_t buflen, unsigned int flags)
 static uint64_t
 get_platform_seed(uint64_t seed)
```

Comment:
On Mon, 14 Sep 2015 07:09:51 -0700, Leandro A. F. Pereira wrote:
> > @@ -86,7 +86,9 @@ getrandom_shim(void *buf, size_t buflen, unsigned int flags)
> >  static uint64_t
> >  get_platform_seed(uint64_t seed)
> 
> Doesn't RIOT have anything to generate some entropy? (Maybe we could test if `RDRAND` is available?)

It has an implementation of the same algorithm we are using, and if the
cpu supports it and it's implemented, it has hardware random support,
but not on the board I'm testing with.
The Arduino Due seems to have one, I can get that board later to
implement that part.